src/main/resources/reference.conf

```diff
@@ -1354,6 +1354,106 @@ resourceTypes = {
     }
     reuseIds = false
   }
+  mct_mapping = {
+    actionPatterns = {
+        delete = {
+            description = "delete this resource"
+        }
+        read_policies = {
+            description = "view all policies and policy details for this resource"
+          }
+          set_parent = {
+            description = "set the parent workspace of this resource"
+          }
+          "share_policy::owner" = {
+            description = "change the membership of the owner policy for this resource"
+          }
+          "share_policy::editor" = {
+            description = "change the membership of the editor policy for this resource"
+          }
+          "share_policy::writer" = {
+            description = "change the membership of the writer policy for this resource"
+          }
+          "share_policy::reader" = {
+            description = "change the membership of the reader policy for this resource"
+          }
+          "read" = {
+            description = "perform reader actions on the resource"
+          }
+          "write" = {
+            description = "perform writer actions on the resource"
+          }
+          "edit" = {
```

Comment:
again, not sure what actions you need here and I'm not sure if you need both `write` and `edit` actions.  These actions can be whatever you want, there are no particular actionPatterns that must be present here, so just make sure you're defining actions you actually care about.  So maybe you do want `write` and `edit`, I have no idea what the product needs are here, just wanted to clarify.

Reply:
same goes for `mct_target`

Reply:
removed edit because it doesn't have a clear use for this resource

Reply:
note: removed the `edit` action because as you point out there's no real distinction here, but left the editor role which is kind of a super-writer with more sharing